### ISSUE TITLE
[Design] Simplify the CRUD view filter controls

### ISSUE THREAD

**Issue by ktmud** [apache/superset#18724]
## Screenshot

<img width="1136" alt="image" src="https://user-images.githubusercontent.com/335541/153955909-bfbddca2-4216-4535-93b1-c1ba78ba6cb9.png">

## Description

Currently Charts and Dashboards page have a lot of filters taking too much space, resulting in double rows in smaller screens. This is not an efficient use of space. There must be some more sophisticated ways of arranging these filters. For example, the boolean filters (favorated or certificated or not) could be replaced with icon toggles.

**Comment by srinify** [comment]
Yeah this is a less than ideal experience. We've hit it a few times internally and have informally discussed what to do about it. But for now -- Jesse, how do you feel about moving this over to Ideas https://github.com/apache/superset/discussions/categories/ideas

@geido @villebro and I have been trying to keep Github issues curated & focused on bugs or bugs to be replicated, etc. 🙏 

Tagging @jess-dillard @rusackas @yousoph who may know more about this

**Comment by ktmud** [comment]
I think it's okay to keep tasks and feature requests in issues. If we want a clean view of bugs, we can use the label filters. At any time, Engineers should always be working on a healthy mix of bugs and new features, instead of just bugs. We have tons of TODOs from the Operational Model project, which may eventually be converted into issues, too.

To me, discussions are more for user questions and big proposals that do not have a clear path forward. If you believe this issue belongs to that category, then feel free to move it.

**Comment by rusackas** [comment]
Heya. After much discussion, we're trying to move feature requests, design discussions, and config Q&A topics over to Discussions, and put them in the right buckets. This is in hopes that we can use Issues more for bugs, and whittle things down to an actionable backlog. The alternative is to dial up stalebot and whittle that way, which would have closed this by now. I think this is still a valid discussion, and don't want to close it, so I'm tempted to move it to an Ideas thread. 

I'm not sure if @kasiazjc and/or @jess-dillard have an action item on this but I'll ping them here for good measure. 

For where we're at, I think this (and other items) should be moved to discussions to stay alive there and get more feedback. When we get the Issues backlog pared down, hopefully, we can revisit how we use Issues and keep things manageable.
